tests/certification/pubsub/azure/servicebus/README.md

```diff
@@ -0,0 +1,41 @@
+# Azure Service Bus Pubsub Certification
+The purpose of this module is to provide tests that certify the Azure Service Bus Queue Input/Output Binding as a stable component.
+
+## Test Plan
+### Certification Tests
+- Verify with single publisher / single subscriber
+   - Run dapr application with 1 publisher and 1 subscriber
+   - Publisher publishes to 2 topics
+   - Subscriber is subscribed to 1 topic
+   - Simulate periodic errors and verify that the component retires on error
+   - Verify that all expected messages were received
+   - Verify that subscriber does not receive messages from the non-subscribed topic
+- Verify with single publisher / multiple subscribers with same consumerID
+   - Run dapr application with 1 publisher and 2 subscribers
+   - Publisher publishes to 1 topic
+   - Subscriber is subscribed to 1 topic
+   - Simulate periodic errors and verify that the component retires on error
+   - Verify that all expected messages were received
+- Verify with single publisher / multiple subscribers with different consumerIDs
+   - Run dapr application with 1 publisher and 2 subscribers
+   - Publisher publishes to 1 topic
+   - Subscriber is subscribed to 1 topic
+   - Simulate periodic errors and verify that the component retires on error
+   - Verify that all expected messages were received
+- Verify with multiple publishers / multiple subscribers with different consumerIDs
+   - Run dapr application with 2 publishers and 2 subscribers
+   - Publisher publishes to 1 topic
+   - Subscriber is subscribed to 1 topic
+   - Simulate periodic errors and verify that the component retires on error
+   - Verify that all expected messages were received
+- Verify data with a topic that does not exist
+   - Run dapr application with 1 publisher and 1 subscriber
+   - Verify the creation of service bus
+   - Send messages to the service created
+   - Verify that subscriber received all the messages
+- Verify reconnection after the network interruptions
```

Comment:
I don't see this part in the test, has it been completed?

Reply:
Added now!